File: 344-ReverseString/344-ReverseString.py
# Last updated: 17/07/2025, 11:26:22
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        for i in range(0,len(s)//2):
            s[i], s[-1-i]= s[-1-i], s[i]
        return s

File: 344-ReverseString/344-ReverseString.java
// Last updated: 17/07/2025, 11:26:27
class Solution {
    public void reverseString(char[] s) {
        for(int i = 0; i < (s.length/2); i++){
            char temp = s[i];
            s[i] = s[s.length-(i+1)];
            s[s.length-(i+1)] = temp;
        }
    }
}